Doc/library/asyncio-task.rst

```diff
@@ -254,6 +282,78 @@ Creating Tasks
       Added the *context* parameter.
 
 
+Task Groups
+===========
+
+Task groups combine a task creation API with a convenient
+and reliable API to wait for all tasks in the group to finish.
```

Comment:
```suggestion
and reliable way to wait for all tasks in the group to finish.
```

Avoid repetitive and potentially confusing repeat of "API", and use plainer and more direct language